javaslang/src/main/java/javaslang/collection/BitMappedTrie.java

```diff
@@ -41,32 +41,32 @@
     @SuppressWarnings("unchecked")
     static <T> BitMappedTrie<T> empty() { return (BitMappedTrie<T>) EMPTY; }
 
-    private final Object[] array;
```

Comment:
intermediary commit, that removes all `Object[]` references, changing them to `Object`s and accessing them in a single place using `getAt`, `setAt` and `lengthOf`, so that these methods can be specialized in the next commit to every primitive.